### ISSUE TITLE
Episode interviewing the RFC shepherds

### ISSUE THREAD

**Issue by carols10cents** [request-for-explanation/podcast#9]
Should we do an episode at some point with some/all of the shepherds as guests and have them talk about their role in the process?

**Comment by Manishearth** [comment]
Yes,but only if we get to grill them on why `$MY_PET_FEATURE` was killed

**Comment by joshtriplett** [comment]
@Manishearth 
> Yes,but only if we get to grill them on why $MY_PET_FEATURE was killed

A survey of "why was this RFC rejected, and where do we go from here" for a half-dozen popular-but-rejected RFCs would make for an interesting episode.
